venv/include/python2.7

```diff
@@ -0,0 +1 @@
+/usr/local/Cellar/python/2.7.14_3/Frameworks/Python.framework/Versions/2.7/include/python2.7
```

Comment:
Use this command `git rm --cached venv/include/python2.7` to ignore the previously committed file and it will be ignored after that.